lib/credential/data/credential_repository_impl.dart

```diff
@@ -53,16 +53,25 @@ class CredentialRepositoryImpl extends CredentialRepository {
   }
 
   @override
-  Future<List<ClaimEntity>> getClaims(
-      {List<FilterEntity>? filters,
-      required String genesisDid,
-      required String privateKey}) async {
+  Future<List<ClaimEntity>> getClaims({
+    List<FilterEntity>? filters,
+    required String genesisDid,
+    required String privateKey,
+    bool sortByExpiration = false,
```

Comment:
Can't we just intoduce new entity `SortOrder` like enum or smthg like that?
And it will have pretty the same values as current params.